releases/latest/beta/Dockerfile.ubi.openjdk17

```diff
@@ -28,8 +28,8 @@ FROM icr.io/appcafe/ibm-semeru-runtimes:open-17-jdk-ubi
 
 USER root
 
-ARG LIBERTY_VERSION=23.0.0.12-beta
-ARG LIBERTY_BUILD_LABEL=cl231120231030-1102
+ARG LIBERTY_VERSION=24.0.0.1-beta
+ARG LIBERTY_BUILD_LABEL=replace_with_gm_driver_label
```

Comment:
Noticed that the placeholder for LIBERTY_SHA has curly brackets, while this one doesn't. Probably doesn't matter, but just wanted to point it out.

Reply:
Hi, Leo. That value comes from the boilerplate text that is prefilled in the "Enter build label of release driver" of the "Run workflow" dialog in the Actions tab. I will fix the workflow file or wherever this "dummy" text is and add the braces.